### PR TITLE
dacap-clip: add recipe

### PR DESCRIPTION
**Pull request by toge**
Specify library name and version:  **dacap-clip/1.5**

dacap/clip is cross-platform clipboard library.

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by prince-chrismc — APPROVED** [review]

**Comment by uilianries** [comment]
@toge please, consider https://github.com/toge/conan-center-index/pull/2. It updated this recipe for Conan 2.0

**Comment by toge** [comment]
@uilianries 
Thank you for yore PR.
I merged it.

**Review by uilianries — APPROVED** [review]

**Review by jgsogo — APPROVED** [review]

**Review by prince-chrismc — DISMISSED** [review]

**Review by uilianries — DISMISSED** [review]

File: recipes/dacap-clip/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True, destination=self.source_folder)
+
+    def generate(self):
+        toolchain = CMakeToolchain(self)
+        toolchain.variables["CLIP_EXAMPLES"] = False
+        toolchain.variables["CLIP_TESTS"] = False
+        toolchain.variables["CLIP_X11_WITH_PNG"] = self.options.get_safe("with_png", False)
+        if is_msvc(self):
+            toolchain.cache_variables["CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS"] = self.options.shared
```
Comment (uilianries) [inline recipes/dacap-clip/all/conanfile.py:71]:
```suggestion
            toolchain.cache_variables["CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS"] = bool(self.options.shared)
```
cache_variables still does not parse options, but there is a PR which should fix it on Conan 1.53

**Comment by prince-chrismc** [comment]
Please add the missing macos framwprks https://github.com/conan-io/conan-center-index/pull/12034#issuecomment-1247041086

**Comment by toge** [comment]
@prince-chrismc 
Thank you for comment.
I am going to solve it.

**Review by uilianries — APPROVED** [review]
LGTM 👌

**Review by prince-chrismc — APPROVED** [review]

**Pull request merged**